### PR TITLE
Fix lint in test_dataloader.py

### PR DESCRIPTION
**Pull request by ezyang**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:white_circle:&nbsp; #17813 Suppress C408 lint (don't use dict constructor)&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D14390136/)
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#17820 Fix lint in test_dataloader.py**&nbsp;&nbsp;[:green_heart:](https://our.internmc.facebook.com/intern/diff/D14392864/)
&nbsp;&nbsp;&nbsp;&nbsp;:white_circle:&nbsp; #17821 Fix lint in test_distributions.py&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D14392899/)
&nbsp;&nbsp;&nbsp;&nbsp;:white_circle:&nbsp; #17823 Fix lint in test_jit.py&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D14392996/)

Differential Revision: [D14392864](https://our.internmc.facebook.com/intern/diff/D14392864/)

### CODE REVIEW

**Review by eellison — COMMENTED** [review]
Thanks for doing this!

File: test/test_dataloader.py [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ def setUp(self):
     def test_custom_batch_pin(self):
         loader = DataLoader(self.dataset, batch_size=2, collate_fn=collate_wrapper,
                             pin_memory=True)
-        for batch_ndx, sample in enumerate(loader):
+        for _sample in loader:
```
Comment (vishwakftw) [inline test/test_dataloader.py]: Minor nit here: `_sample` -> `sample`

**Comment by vishwakftw** [comment]
@ezyang The nit seems to have broken tests on master.

**Comment by ezyang** [comment]
Crap, reverting

**Pull request closed without merging**